WebRoot/js/zimbraMail/calendar/view/ZmApptQuickAddDialog.js

```diff
@@ -85,42 +77,25 @@ function() {
 	return "ZmApptQuickAddDialog";
 };
 
-
-ZmApptQuickAddDialog.prototype.getFreeBusyCache =
-function() {
-    return this._fbCache;
-}
-
 ZmApptQuickAddDialog.prototype.initialize = 
 function(appt) {
 	this._appt = appt;
 
+	this.setTitle(AjxMessageFormat.format(ZmMsg.quickAddAppt, [DwtCalendar.getDateFormatter().format(appt.startDate)]));
+
 	// reset fields...
 	this._subjectField.setValue(appt.getName() ? appt.getName() : "");
-	this._locationField.setValue(appt.getLocation() ? appt.getLocation() : "");
-	this._startDateField.value = AjxDateUtil.simpleComputeDateStr(appt.startDate);
-	this._endDateField.value = AjxDateUtil.simpleComputeDateStr(appt.endDate);
 	var isAllDay = appt.isAllDayEvent();
+	// @TODO what to do when all day?
```

Comment:
In all day, I guess time selectors are hidden as per requirement.

Are we expecting anything more? I mean is this `TODO` still valid?

Reply:
removed